src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,52 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
+
+public class Lotto {
+    private static final String NUMBER_COUNT_ERROR = "[ERROR] 6개의 숫자를 입력해주세요";
+    private static final String NUMBER_DUPLICATE_ERROR = "[ERROR] 숫자는 중복될 수 없습니다";
+    private static final String NUMBER_RANGE_ERROR = "[ERROR] 1 ~ 45 사이의 숫자를 입력해주세요";
+    private static final int MAXIMUM_NUMBER = 45;
+    private static final int MINIMUM_NUMBER = 1;
+    private static final int LOTTO_NUMBER_LIMIT = 6;
+    private final ArrayList<Integer> lottoNumbers;
+
+    public Lotto(ArrayList<Integer> nums) {
+        validateCount(nums);
+        validateDuplicate(nums);
+        validateNumsRange(nums);
+        this.lottoNumbers = nums;
+    }
+
+    public ArrayList<Integer> getLottoNumbers() {
+        return this.lottoNumbers;
+    }
+
+    private void validateCount(ArrayList<Integer> nums) {
+        if (nums.size() != LOTTO_NUMBER_LIMIT)
+            throw new IllegalArgumentException(NUMBER_COUNT_ERROR);
+    }
+
+    private void validateDuplicate(ArrayList<Integer> nums) {
+        Set<Integer> numbers = new HashSet<>();
+        numbers.addAll(nums);
+        if (nums.size() != numbers.size())
+            throw new IllegalArgumentException(NUMBER_DUPLICATE_ERROR);
+    }
+
+    private void validateNumsRange(ArrayList<Integer> nums) {
+        for (int i = 0; i < nums.size(); i++)
+            validateNumRange(nums.get(i));
```

Comment:
```suggestion
        for (int i = 0; i < nums.size(); i++) {
            validateNumRange(nums.get(i));
        }
```
아래 링크 `7. 2 복합문` 부분 참고 부탁드릴게요! 
https://myeonguni.tistory.com/1596

Reply:
헉 중괄호에도 컨벤션이 있는지 몰랐네요..! 😂
컨벤션 지켜 수정하겠습니다!